src/components/chips/chips.scss

```diff
@@ -0,0 +1,28 @@
+md-chips {
+  md-chips-wrap {
+    li {
+      border: 1px solid #DCDCDC;
+      border-radius: 11px;
+      background: #F5F5F5;
+      padding-left:5px;
+      padding-top: 2px;
+      margin-left: 4px;
+      display: inline-block;
+      color: rgba(0,0,0,0.8);
+      font-family: roboto;
+      font-size: 13px;
+      height:22px;
```

Comment:
Lots of magic-value numbers in here. Can you extract some into SCSS variables with some semantically meaningful name?